File: TYPESCRIPT/Cap05/c05ex03.ts
// c05ex03.ts

import * as rls from 'readline-sync';

let I: number = 1;
while (!(I > 5)) {
  let N: number = rls.questionInt('Entre um valor numerico: ');

  let R: number = N * 3;

  console.log("Resultado = " + R + "\n");

  I++;
}

console.log("");
process.stdout.write('Tecle <Enter> para encerrar... ');
rls.question();

File: VB_NET/Cap05/c05ex03.vb
' c05ex03.vb

Imports System
Imports System.Console

Module Program
    Sub Main(args As String())
        Dim I As Integer
        Dim N As Integer
        Dim R As Integer

        I = 1
        Do Until (I > 5)
            Write("Entre um valor numerico: ")
            N = Integer.Parse(ReadLine())

            R = N * 3

            WriteLine("Resultado = " & R & vbCrLf)

            I += 1
        Loop

        Write(vbCrLf & "Tecle <Enter> para encerrar... ")
        ReadLine()
    End Sub
End Module
